### ISSUE TITLE
Wiederherstellung des letzten Zustands

### ISSUE THREAD

**Issue by wis36** [minova-afis/aero.minova.rcp#1404]
Ich bekomme beim Start diese Meldung:

![grafik](https://user-images.githubusercontent.com/20420949/207533547-e9751af9-b1cf-4d8f-aebe-6bacb9be9199.png)

Die Meldung kann man deaktivieren, aber die Funktionalität bleibt wahrscheinlich gleich:
Obwohl ich eine Standard-Einstellung gespeichert habe, lädt das WFC nicht diese, sondern den "letzten Zustand".

MMn sollte der letzte Zustand nur wiederhergestellt werden, falls kein Standard / DEFAULT gespeichert ist.

**Comment by janiak-minova** [comment]
Das müssen wir definieren. Ich persönlich finde den letzten Zustand komfortabler, da muss man nicht daran denken, den Zustand zu speichern.

Es wäre auch denkbar, eine Einstellung zu erstellen.

**Comment by wis36** [comment]
Ja, finde ich auch komfortabel. War auch im 10er so, in der NatTable hatte ich es dann irgendwie nicht mehr implementiert.

Ich weiß nicht, ob man den Unterschied braucht zwischen
* Standard = wird beim Start automatisch geladen
* Default = wird beim Drücken von "Laden" geladen, wenn nicht angegeben wurde, welchen Zustand

**Comment by janiak-minova** [comment]
<img width="174" alt="Bildschirmfoto 2023-01-02 um 09 41 24" src="https://user-images.githubusercontent.com/77741125/210209390-327c3514-97b2-429d-b0f4-adf5dc6dc3db.png">

Der Unterschied wird tatsächlich schon gemacht. Wenn man auf Speichern oder Laden direkt klickt, wird `DEFAULT` angelegt/geladen.

Der letzte Zustand wird intern gespeichert (`LAST_STATE`), ist aus der Anwendung aber gar nicht auswählbar.

**Comment by wis36** [comment]
Das meinte ich eben 😉
